src/sql/workbench/services/accountManagement/browser/accountPickerImpl.ts

```diff
@@ -86,36 +96,66 @@ export class AccountPicker extends Disposable {
 	public createAccountPickerComponent() {
 		// Create an account list
 		const delegate = new AccountListDelegate(AccountPicker.ACCOUNTPICKERLIST_HEIGHT);
+		const tenantDelegate = new TenantListDelegate(AccountPicker.ACCOUNTPICKERLIST_HEIGHT);
+
 		const accountRenderer = new AccountPickerListRenderer();
+		const tenantRenderer = new TenantPickerListRenderer();
+
 		this._listContainer = DOM.$('div.account-list-container');
-		this._accountList = new List<azdata.Account>('AccountPicker', this._listContainer, delegate, [accountRenderer]);
+		this._tenantListContainer = DOM.$('div.tenant-list-container');
+
+		this._accountList = new List<azdata.Account>('AccountPicker', this._listContainer, delegate, [accountRenderer], {
+			setRowLineHeight: false,
+		});
+		this._tenantList = new List<Tenant>('TenantPicker', this._tenantListContainer, tenantDelegate, [tenantRenderer]);
```

Comment:
This all seems like duplicated stuff from above. Why not make a common widget that contains both and sets up all this stuff?